Plots/Trajectory/NCL_polyg_14.py

```diff
@@ -0,0 +1,96 @@
+"""
+NCL_polyg_14.py
+================
+This script illustrates the following concepts:
+    - Drawing polylines and markers using great circle paths
+    - Using gc_latlon to calculate a great circle path
```

Comment:
Update your comments to reflect what this example does. `gc_latlon` is from NCL

Reply:
Updated in most recent push

Reply:
Just kidding, pushed the wrong notebook!